core/core-dotspacemacs.el

```diff
@@ -455,17 +455,19 @@ Called with `C-u C-u' skips `dotspacemacs/user-config' _and_ preleminary tests."
                 (load-file buffer-file-name)
                 (dotspacemacs|call-func dotspacemacs/init
                                         "Calling dotfile init...")
-                (dotspacemacs|call-func dotspacemacs/user-init
-                                        "Calling dotfile user init...")
+                (let (spacemacs--keybinding-collision-check nil)
```

Comment:
The let binding is wrong, either `(let (spacemacs--k-c-c) ...)` or `(let ((spacemacs--k-c-c nil)) ...)` (note the amount of parenthesis)